queries.sql

```diff
@@ -8,3 +8,64 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Transactions
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
* Kindly verify that the change was made after `species = unspecified` and verify that the species columns went back to the state before the transaction.
 ![image](https://user-images.githubusercontent.com/52497246/207435729-983980bd-510a-4292-af5a-23e563a6981a.png)